### PR TITLE
Fix for AmazonQ on Linux input focus problem

### PR DESCRIPTION
**Pull request by ppiliaev**
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

## Description
Fix for the issue
https://github.com/aws/aws-toolkit-jetbrains/issues/4100

Implemented according to IntelliJ IDEA guideline:

- https://youtrack.jetbrains.com/issue/IDEA-241671/JCEF-focus-issues
- https://youtrack.jetbrains.com/issue/JBR-5083

## Checklist
- [x] My code follows the code style of this project
- [ ] I have added tests to cover my changes
- [ ] A short description of the change has been added to the **[CHANGELOG](https://github.com/aws/aws-toolkit-jetbrains/blob/master/CONTRIBUTING.md#contributing-via-pull-requests)** if the change is customer-facing in the IDE.
- [ ] I have added metrics for my changes (if required)
 
## License
I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by manodnyab** [comment]
Can you please add a changelog for this?

**Comment by rli** [comment]
I believe we previously disabled OSR due to a flickering issue when multiple JCEF panels were open at the same time. Can you confirm that this is no longer an issue? (For example, what happens the markdown rendered preview panel and the Q panel are on screen together?)

**Comment by ppiliaev** [comment]
> I believe we previously disabled OSR due to a flickering issue when multiple JCEF panels were open at the same time. Can you confirm that this is no longer an issue? (For example, what happens the markdown rendered preview panel and the Q panel are on screen together?)

It works fine now:

Screenshot 2024-02-13 at 18 20 09

File: .changes/next-release/bugfix-6ce9e38b-9d86-4273-a47a-34595c4ef2a1.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "bugfix",
+  "description" : "Fix for AmazonQ on Linux input focus problem https://github.com/aws/aws-toolkit-jetbrains/issues/4100"
```
Comment (rli) [inline .changes/next-release/bugfix-6ce9e38b-9d86-4273-a47a-34595c4ef2a1.json:3]:
```suggestion
  "description" : "Fix for AmazonQ on Linux input focus problem (#4100)"
```
automation automatically turns this into links

Comment (ppiliaev) [reply .changes/next-release/bugfix-6ce9e38b-9d86-4273-a47a-34595c4ef2a1.json]: Done

**Review by rli — APPROVED** [review]

**Review by barbepatAWS — APPROVED** [review]

**Pull request merged**